backend/src/main/resources/graphql/main.graphqls

```diff
@@ -288,9 +292,33 @@ type TestResultsPage {
   content: [TestResult]
 }
 
+type Result {
+  id: ID!
+  facility: Facility!
+  patient: Patient!
+  dateAdded: String!
+  dateUpdated: DateTime
+  pregnancy: String
+  noSymptoms: Boolean!
+  symptoms: String!
+  symptomOnset: LocalDate
```

Comment:
do we need AOE answers on the results object? (pregnancy, noSymptoms, symptoms, symptomOnset)

Reply:
Nah, removed in https://github.com/CDCgov/prime-simplereport/pull/6682/commits/34346c5a58c35cca7a4fde47695fcf9e994ce8ae.